parachain/runtimes/gargantua/src/lib.rs

```diff
@@ -726,6 +753,8 @@ construct_runtime!(
 		Fishermen: pallet_fishermen = 61,
 		TokenGatewayInspector: pallet_token_gateway_inspector = 62,
 		IsmpSyncCommitteeGno: ismp_sync_committee::pallet::<Instance2> = 63,
+		TokenGateway: pallet_token_gateway = 64,
+		HyperBridge: pallet_hyperbridge = 65,
```

Comment:
Lets revert, hyperbridge doesn't use pallet-token-gateway. This is for downstream crates

Reply:
done